website_sale_templates_view_adj/views/website_sale_templates_view_adj.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <template id="website_sale_templates_view_adj" inherit_id="website_sale.payment_sale_note">
+        <xpath expr="//div[@id='payment_method'][hasclass('js_payment')]" position="after">
+            <div class="custom-control custom-checkbox float-right mt-2 oe_accept_cgv_button">
+                <input type="checkbox" id="checkbox_cgv" class="custom-control-input"/>
+                <label for="checkbox_cgv" class="custom-control-label">
+                    We agree to the <a target="_BLANK" href="/shop/terms">terms &amp; conditions</a>
+                    <br/>
+                    according to the current agreement.
+                </label>
+            </div>
+        </xpath>
```

Comment:
```suggestion
        <xpath expr="//label[hasclass('custom-control-label')]" position="after">
            <span>
                according to the current agreement.
            </span>
        </xpath>
```

Note that this style of embedding tags in the middle of a sentence makes the translation tricky. It should be OK this time as it's intended to be enable for the overseas website (i.e. English) only.